include/clang/Constraints/Constraints.h

```diff
@@ -0,0 +1,457 @@
+//                     The LLVM Compiler Infrastructure
```

Comment:
I think it'd be a good to move them into the folder with the rest of the code.    I'm trying to minimize changes to clang to make moving Checked C changes upstream easier later.